src/main/kotlin/uk/gov/dluhc/printapi/rest/TemporaryCertificateController.kt

```diff
@@ -35,9 +40,18 @@ class TemporaryCertificateController(
     fun getTemporaryCertificateSummariesByApplicationId(
         @PathVariable eroId: String,
         @PathVariable applicationId: String,
-    ): TemporaryCertificateSummariesResponse {
-        TODO("not yet implemented")
-    }
+    ): TemporaryCertificateSummariesResponse =
+        TemporaryCertificateSummariesResponse(
+            temporaryCertificateSummaryService.getTemporaryCertificateSummaries(
+                eroId,
+                SourceType.VOTER_CARD,
+                applicationId
+            ).map {
+                temporaryCertificateSummaryMapper.toApiTemporaryCertificateSummary(it)
+            }.sortedByDescending {
+                it.dateTimeGenerated
```

Comment:
seems like a UI concern?

Reply:
Yes and no ....
For most of our APIs (in all our apps) that return a collection of things where a date is important, the UI have asked that we return them in a sorted order. It helps them; and I kinda get it.
Sure, its a UI concern .... but it also feels like an API concern in that it should be part of our implementation to return things in a given sort order.
I've implemented the sorting here at the controller - its the closest point to the UI, so it feels like the right approach?

Reply:
Agree - I liked that it's in the controller (rather than in the service layer) 👍